src/firefly/js/api/ApiUtil.js

```diff
@@ -73,6 +72,19 @@ export function renderDOM(div, Component, props, style={width:'100%', height:'10
 }
 
 
+export function myrenderDOM(div, Component, props) {
+    const divElement= isString(div) ? document.getElementById(div) : div;
+
+    if (!isElement(divElement)) debug(`the div element ${isString(div)?div:''} is not defined in the html` );
+    if (!Component) debug('Component must be defined');
+
+
+    const renderStuff= (
+            <Component {...props} />
+    );
+
+    ReactDOM.render(renderStuff,divElement);
+}
```

Comment:
Is there something different you want to do here that's not in renderDOM?  If it can be applied to renderDOM, it should be combine into it.